### PR TITLE
CLN: Collapse private ._take implementation into the public take method

### LINKED ISSUE #27174: CLN: Collapse private ._take implementation into the public take method
xref https://github.com/pandas-dev/pandas/pull/27171#issuecomment-507505074

An internal `._take` method was created to process a keyword argument deprecation.
Now that both `._take` and `.take` have the same signature, the code within `._take` can be collapsed into the public `.take` method.

https://github.com/pandas-dev/pandas/blob/527e714647a16ad31f80ea535af5a63156515861/pandas/core/generic.py#L3322

### PR DESCRIPTION
**Pull request by endenis**
- [x] closes #27174
- [ ] tests added / passed
- [X] passes `black pandas`
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

Getting rid the `_take` private method that seems to be no longer needed.

### CODE REVIEW

File: pandas/core/series.py [hunk 1]
```diff
@@ -4371,8 +4371,9 @@ def memory_usage(self, index=True, deep=False):
             v += self.index.memory_usage(deep=deep)
         return v
 
-    @Appender(generic.NDFrame._take.__doc__)
-    def _take(self, indices, axis=0, is_copy=False):
+    @Appender(generic.NDFrame.take.__doc__)
+    def take(self, indices, axis=0, is_copy=False, **kwargs):
```
Comment (WillAyd) [inline pandas/core/series.py]: Do we need to add kwargs here or did you just add this for consistency?

**Review by WillAyd — COMMENTED** [review]
Thanks for the PR! @mroeschke

File: pandas/core/series.py (hunk 1 above)
Comment (endenis) [reply pandas/core/series.py]: I added it for consistency:
`Series` class inherits from the `NDFrame`, so I wanted to preserve the same signature for `take`.

**Comment by jreback** [comment]
thanks @endenis

**Pull request merged**